kotlin/853-Car-Fleet.kt

```diff
@@ -0,0 +1,27 @@
+package kotlin
```

Comment:
You can remove this package - don't need it in Leetcode

Reply:
Sure